ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
+  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
+  exit 1
+fi
+
+# Set PATH to include YQ, installed via pip in the image
+export PATH="$PATH:/usr/local/bin"
+
+CLUSTER_NAME=${NAMESPACE}-${JOB_NAME_HASH}
+RESOURCE_GROUP=${CLUSTER_NAME}
+AZURE_AUTH_LOCATION="${SHARED_DIR}/osServicePrincipal.json"
+APP_ID=$(jq -r .clientId "${AZURE_AUTH_LOCATION}")
+TENANT_ID=$(jq -r .tenantId "${AZURE_AUTH_LOCATION}")
+AAD_CLIENT_SECRET=$(jq -r .clientSecret "${AZURE_AUTH_LOCATION}")
+SUBSCRIPTION_ID=$(jq -r .subscriptionId "${AZURE_AUTH_LOCATION}")
+AZURESTACK_ENDPOINT=$(cat "${SHARED_DIR}"/AZURESTACK_ENDPOINT)
```

Comment:
Isn't this already being set in the ref.yaml file here?

>- name: AZURESTACK_ENDPOINT
    default: 'https://management.ppe3.stackpoc.com'

Reply:
Yes! I removed the vars from the ref.yaml file, I think removing from that file is more in line with the subsequent goal of reading this info from the lease.

Reply:
Yeah makes sense.